packages/quantic/force-app/main/default/lwc/quanticQuickviewContent/quanticQuickviewContent.js

```diff
@@ -0,0 +1,47 @@
+import {LightningElement, api, track} from 'lwc';
+// @ts-ignore
+import defaultTemplate from './quanticQuickviewDefault.html';
+// @ts-ignore
+import youtubeTemplate from './quanticQuickviewYoutube.html';
+
+/** @typedef {import("coveo").Result} Result */
+
+const allowedHost = ['https://youtube.com/'];
+
+export default class QuanticQuickviewContent extends LightningElement {
```

Comment:
The js doc to document the component would be nice! take the other components as a reference.
Also this I think this component can be tagged as `@internal` since it's really an internal component and we don't really want it to appear in the Quantic documentation.